packages/neutron-sdk/src/bindings/msg.rs

```diff
@@ -27,6 +27,14 @@ pub struct IbcFee {
     pub timeout_fee: Vec<Coin>,
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
+pub enum ChannelOrdering {
+    None,
```

Comment:
Seems like it should be  `ORDER_NONE_UNSPECIFIED`  and if you set Ordering to `None` variant you get the error  https://github.com/neutron-org/neutron/blob/b3a0f5d2a31d37a8ac91e81308762f8fc39ea4f3/wasmbinding/message_plugin.go#L877 because there is no `NONE` in a values map.